hazelcast/src/main/java/com/hazelcast/nearcache/NearCacheStats.java

```diff
@@ -25,108 +25,81 @@
 public interface NearCacheStats extends LocalInstanceStats {
 
     /**
-     * Returns the creation time of this Near Cache on this member.
-     *
      * @return creation time of this Near Cache on this member
```

Comment:
Minor: maybe make this explicit that the time returned is millis since epoch?

Reply:
> Minor: maybe make this explicit that the time returned is millis since epoch?

I agree, but that pattern is repeated in lots of places (e.g. `com.hazelcast.core.EntryView.getCreationTime()`, `com.hazelcast.core.EntryView.getCreationTime()`).

Maybe this could be addressed in a separate PR?